bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -1485,6 +1485,9 @@ public boolean isOutputLevelFixed() {
                 case "84934718":
                     codec = "PCM51";
                     break;
+                case "89434721":
+                    codec = "DTSSurround51";
```

Comment:
I will rather call it DTS5.1.
I see an elready existing codec for DTS. Is it for DTS 2.0 ?  Please change it if this is the case in order to better distinguish with your added entry. 
I am a little surprised we are not using space in certain vodec but that is another story.

Reply:
The sonos app showed it as DTS Surround 5.1 when I checked it.   I'm not sure what the other one is, I'll have to go dig out the old PR.  Same with the spaces, I have a vague memory we did that on purpose but don't remember why.

Reply:
I see now. We did it this way and put the longer name in channels.xml. I'll update the PR.

Reply:
I added the line to channels.xml and adjusted it to just be DTS51.  I went back to #9997 to see what the original DTS with the ID of 32 was from and it looks like it was identified by @stjx79 in the original request.  I'm not sure if that pertains to DTS 2.0 or not.  To note, it is possible this could be DTS MA which may be the distinguishing factor.  The source is DTS MA, but it goes through my receiver which may change it, I'm not entirely sure.

Reply:
Do you mean DTS-HD MA? You mean "89434721" could actually be that and not DTS 5.1?

Reply:
Yes. From the perspective of the Sonos app, it shows as "DTS Surround 5.1".  The actual media is DTS-HD MA 7.1.  I believe my Denon is changing it so I can't be 100% sure what's actually on the wire going into the Sonos.

Reply:
I was myself mixing-up the codec name we defined and the label for this codec.